pkg/harvester/edit/harvesterhci.io.host/index.vue

```diff
@@ -398,16 +406,50 @@ export default {
     },
 
     async saveLonghornNode() {
+      const inStore = this.$store.getters['currentProduct'].inStore;
+
       const disks = this.longhornNode?.spec?.disks || {};
 
       this.newDisks.map((disk) => {
         (disks[disk.name] || {}).tags = disk.tags;
       });
 
-      try {
-        await this.longhornNode.save();
-      } catch (err) {
-        return Promise.reject(exceptionToErrorsArray(err));
+      let count = 0;
+
+      const retrySave = async() => {
+        try {
+          await this.longhornNode.save();
+        } catch (err) {
+          if ((err.status === 409 || err.status === 403) && count < 3) {
+            count++;
+
+            await this.$store.dispatch(`${ inStore }/find`, {
+              type: LONGHORN.NODES,
+              id:   this.longhornNode.id,
+              opt:  { force: true },
+            });
+
```

Comment:
We can use this [method](https://github.com/harvester/dashboard/blob/bd11739fa3257f74c698e65f122f3d6f7b6df975/shell/mixins/create-edit-view/impl.js#L118) to resolve conflicts.

Reply:
I don't think we have to use that method, I think use this [method](https://github.com/rancher/dashboard/blob/dcc4a73e498df7455ff292ffad46b2781cb81a7a/shell/models/management.cattle.io.project.js#L104) also make sense.